src/API/Google/AdsAssetGroupAsset.php

```diff
@@ -121,4 +133,84 @@ public function get_assets_by_asset_group_ids( array $asset_groups_ids ): array
 		}
 
 	}
+
+	/**
+	 * Edit assets group assets.
+	 *
+	 * @param int   $asset_group_id The asset group id.
+	 * @param array $assets The assets to create.
+	 *
+	 * @return array The asset group asset operations.
```

Comment:
💅  Seems like `create_operation` could potentially throw an extension which is not handled neither added to the phpdocs.

Reply:
I am not sure about this, what I can see the `MutateOperation` or any of their methods `setAssetGroupAssetOperation `or `setCreate`, would not throw an exception. The exception will throw when the mutating operations are executed in this line:  https://github.com/woocommerce/google-listings-and-ads/blob/1d9f8deda7f031738c8fa457948a65fae99a7ece/src/API/Google/AdsAssetGroup.php#L311

Reply:
What I meant is that inside `$this->asset->create_operation(`) we throw an exception in the default method. 

```
default:
			throw new Exception( 'Asset type not supported' );
```

This exception is not being handled inside `edit_operation` method. So either we handled that exception inside `edit_operation` or we can just add  `* @throws Exception` in PHPDOC